code/framework/src/scripting/engines/node/builtins/color_rgba.h

```diff
@@ -0,0 +1,118 @@
+/*
+ * MafiaHub OSS license
+ * Copyright (c) 2021-2023, MafiaHub. All rights reserved.
+ *
+ * This file comes from MafiaHub, hosted at https://github.com/MafiaHub/Framework.
+ * See LICENSE file in the source repository for information regarding licensing.
+ */
+
+#pragma once
+
+#include "../../../keys.h"
```

Comment:
Let's change it to scripting/keys.h include, same for the others.

Reply:
Removed since: https://github.com/MafiaHub/Framework/pull/93